alembic/versions/43be265c03c0_auto_report_allow_bots.py

```diff
@@ -0,0 +1,31 @@
+"""auto_report_allow_bots migration
+
+Revision ID: 43be265c03c0
+Revises: 94c30396a122
+Create Date: 2024-05-29 08:17:31.278493
+
+"""
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '43be265c03c0'
+down_revision = '94c30396a122'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.get_context().autocommit_block():
```

Comment:
Why do we need to drop the default?

Reply:
apologies, that was my mistake (had to merge quick to fix production last night)

Reply:
Yeah I was just doing my usual and copying the migra.sql in there. It seems to pass but feel free to do something different of you think it is better.

Reply:
trying to fix it now. Because that migration won't break our Fritz (that has already been migrated so the column is there) but could break something like Icare that hasn't been updated yet. Caus effectively thing goes back to the incorrect migration, which works if the table is empty but breaks the migration server otherwise